lms/static/js/i18n/eo/djangojs.js

```diff
@@ -137,7 +137,7 @@
     "<%= user %> has been successfully added to the exception list. Click Generate Exception Certificate below to send the certificate.": "<%= user %> h\u00e4s \u00df\u00e9\u00e9n s\u00fc\u00e7\u00e7\u00e9ssf\u00fcll\u00fd \u00e4dd\u00e9d t\u00f6 th\u00e9 \u00e9x\u00e7\u00e9pt\u00ef\u00f6n l\u00efst. \u00c7l\u00ef\u00e7k G\u00e9n\u00e9r\u00e4t\u00e9 \u00c9x\u00e7\u00e9pt\u00ef\u00f6n \u00c7\u00e9rt\u00eff\u00ef\u00e7\u00e4t\u00e9 \u00df\u00e9l\u00f6w t\u00f6 s\u00e9nd th\u00e9 \u00e7\u00e9rt\u00eff\u00ef\u00e7\u00e4t\u00e9. \u2c60'\u03c3\u044f\u0454\u043c \u03b9\u03c1\u0455#", 
```

Comment:
I'm doubting that these files are meant to be checked in too but I'm not sure....  Maybe @dianakhuang or @nedbat  knows?